httpbakery/client.go

```diff
@@ -125,6 +125,28 @@ func DischargeAll(m *macaroon.Macaroon, client *http.Client, visitWebPage func(u
 	return bakery.DischargeAll(m, ctxt.obtainThirdPartyDischarge)
 }
 
+// GetPublicKey returns the public key provided by a server which is using
+// httpbakery.AddDischargeHandler.
+func GetPublicKey(url string) (*bakery.PublicKey, error) {
```

Comment:
// PublicKeyForLocation returns the public key from a macaroon
// discharge server running at the given location URL.
// Note that this is insecure if an http: URL scheme is used.

?

I think we should probably pass the http client in here too,
so that we're free to use httpbakery.Do in the future if
we want.